### ISSUE TITLE
Attack Heli Bug

### ISSUE THREAD

**Issue by Kolmain** [Kolmain/Evolution-A3#55]
Sir,

First - THANK YOU for the Evo A3 Mission - 
Second - (June 4, 2015) I downloaded the latest version, installed it into my dedicated Arma 3 server and ran it. About 1 hour into the mission (first target area) (nice fight by the way); about 30, 40 or more attack heli's appeared over the target area - it was crazy.

I thought maybe it was just a random occurrence, so I uninstalled that mission, (June 5, 2015) re-downloaded Evolution A3 again, reinstalled it and ran it again - the same thing happened again; so I thought I would report this to you.

The only mod I was running was my admin mod - NSS_AC.

I have no error logs to attach - sorry!

Thank's,

CJHammer

**Comment by Kolmain** [comment]
@CJHammer

**Comment by Kolmain** [comment]
I haven't seen or head of anything like this @CJHammer, has anyone else? @Call-911 ? @griffz? @lampelampelampe ?

**Comment by Call-911** [comment]
30-40 Kajmans..oh lord..:D I'd probably just pee myself right there :D

**Comment by CJHammer** [comment]
Sir,

I got a chance to run the EVO A3 again this morning. Same thing happened. This time, I attached .jpg's for your viewing. It MUST be something on my end as no one else seems to have the issue.

I will let you know when I find the bug.  

Apologies for any problems.

Thanks,

CJHammer
![untitled](https://cloud.githubusercontent.com/assets/12757599/8034251/f30c29ac-0db3-11e5-821a-7e54b7099861.jpg)
![untitled2](https://cloud.githubusercontent.com/assets/12757599/8034256/00314f7c-0db4-11e5-9599-cb1e8d5b332d.jpg)
